sw/device/lib/dif/dif_rstmgr.h

```diff
@@ -196,6 +210,58 @@ dif_rstmgr_result_t dif_rstmgr_reset_info_get(
 DIF_WARN_UNUSED_RESULT
 dif_rstmgr_result_t dif_rstmgr_reset_info_clear(const dif_rstmgr_t *handle);
 
+/**
+ * Enables or disables the alert crash dump capture.
+ *
+ * When enabled, will capture the alert crash dump prior to a triggered reset.
+ *
+ * @param handle A Reset Manager handle.
+ * @param state The new toggle state for the crash dump capture.
+ * @return The result of the operation.
+ */
+DIF_WARN_UNUSED_RESULT
+dif_rstmgr_result_t dif_rstmgr_alert_info_set_enabled(
+    const dif_rstmgr_t *handle, dif_rstmgr_toggle_t state);
+
+/**
+ * Retrieves the size of the alert info crash dump.
+ *
+ * The corresponding registers are always-on and are expected to retain
+ * the information after all type of resets except Power-On-Reset (POR).
+ *
+ * @param handle A Reset Manager handle.
+ * @param count The size of alert info crash dump in number of
+ *              `dif_rstmgr_alert_dump_value_t`.
+ * @return The result of the operation.
+ */
+DIF_WARN_UNUSED_RESULT
+dif_rstmgr_result_t dif_rstmgr_alert_info_dump_get_size(
+    const dif_rstmgr_t *handle, size_t *count);
+
+/**
+ * Alert info crash dump 32-bit value.
+ */
+typedef uint32_t dif_rstmgr_alert_info_dump_value_t;
+
+/**
+ * Reads a single alert info crash dump `value` at `index`.
+ *
+ * Alert info crash dump consists of one or more
+ * `dif_rstmgr_alert_info_dump_value_t`. The total count can be found through
+ * using the `dif_rstmgr_alert_info_dump_get_size`.
```

Comment:
Could you proofread this? I think you dropped a `function` at the end of this paragraph, and the wording seems odd.